core/shared/utils/bh_atomic.h

```diff
@@ -101,6 +112,62 @@ typedef uint16 bh_atomic_16_t;
 #endif
 #endif
 
+#if BH_ATOMIC_64_IS_ATOMIC != 0
+
+#define BH_ATOMIC_64_LOAD(v) __atomic_load_n(&(v), __ATOMIC_SEQ_CST)
```

Comment:
BH_ATOMIC_64_XXX macros are not used in this PR, not sure whether we should change this file? Had better add them when needed?